xapian-core/weight/bm25weight.cc

```diff
@@ -48,8 +60,8 @@ BM25Weight::init(double factor)
     Xapian::doccount tf = get_termfreq();
 
     double tw = 0;
-    if (get_rset_size() != 0) {
-	Xapian::doccount reltermfreq = get_reltermfreq();
+    if (get_rset_size() != 0 && !bm25+) {
```

Comment:
`bm25+` was a parameter to one of the constructors, not a member, so you can't access it here. You probably want to make a `bm25_plus` private member, and initialise that somehow.